### PR TITLE
Fixes #13936 - Drop support for legacy browsers

### PR DESCRIPTION
**Pull request by tbrisker**
This patch removes various workarounds needed to get legacy browsers to
work. Mostly stuff needed for IE<10, but also some other vendor prefixes
that are no longer needed.

### CODE REVIEW

**Comment by ohadlevy** [comment]
I didn't look, but with stats like: `+22 −99,102` lines changed you got the MVP commit of the day :100:

**Comment by tbrisker** [comment]
@ohadlevy I somehow need to get rid of the +98k I got when merging the AllBullets.svg so I can go back to having a net negative impact :laughing:

**Comment by lzap** [comment]
I was wondering if we could implement a warning banner when any of the unsupported browser is in use as part of this patch.

**Comment by tbrisker** [comment]
We already have one for IE<=9, do you think we also need one for other?

File: app/assets/stylesheets/typography.scss [hunk 1]
```diff
@@ -1,10 +1,7 @@
 @font-face {
   font-family: 'Overpass';
   src: local("Overpass"),
-  font-url('overpass_regular-web.eot?#iefix') format("eot"),
-  font-url('overpass_regular-web.woff') format("woff"),
-  font-url('overpass_regular-web.ttf') format("truetype"),
-  font-url('overpass_regular-web.svg#webfontLTZe4IYH') format("svg");
+       font-url('overpass_regular-web.woff') format("woff");
```
Comment (tbrisker) [inline app/assets/stylesheets/typography.scss:4]: All modern browsers support woff, no need to ship other (heavier) formats.

**Comment by dLobatog** [comment]
Merged as bbefa32cd3908d08712a78588a5cf2c04130791c, thanks @tbrisker!

**Pull request closed without merging**